www/js/recommendations.js

```diff
@@ -19,6 +19,19 @@ Ask Shankari about this Wednesday
 
 .controller('RecommendationsCtrl', function(CommHelper, $state, $ionicLoading, $scope, $rootScope, $ionicModal, nzTour,
                                 $window, $http, $ionicPopup, $timeout, storage, ReferralHandler, ReferHelper, Logger, $cordovaInAppBrowser, SurveyLaunch) {
+
+    $scope.currentDisplay = "";
+    //Check and see if this is right with Vanessa
+    //x = window.cordova.plugins.BEMServerComm.pushGetJSON("/suggestion_sys", msgFiller, resolve, reject);
+    //for now using sample
+    $http.get('json/sampleRecommendation.json').then(function(result) {
+        var name = JSON.stringify(result.data.name);
+        var mode = JSON.stringify(result.data.mode);
+        document.getElementById("message_1").innerHTML = name.replace(/\"/g, "");
```

Comment:
This is the wrong way to set HTML values in angular.
It is slow and ugly. @trevorwu's approach, with the `{{}}` is the right approach.